llvm/lib/ProfileData/MemProf.cpp

```diff
@@ -637,6 +638,10 @@ computeFrameHistogram(llvm::MapVector<CallStackId, llvm::SmallVector<FrameIdTy>>
 template llvm::DenseMap<FrameId, FrameStat> computeFrameHistogram<FrameId>(
     llvm::MapVector<CallStackId, llvm::SmallVector<FrameId>>
         &MemProfCallStackData);
+template llvm::DenseMap<LinearFrameId, FrameStat>
```

Comment:
Add a comment that the only thing different is FrameId LinearFrameId (both of which are the same underlying type).

Reply:
They aren't the same underlying type, which is why the templatization was needed. One is uint32_t (LinearFrameId) and one is uint64_t (FrameId).